### PR TITLE
added support for Raven JS

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

**Comment by szok** [comment]
:ok:

File: src/ralph/admin/templatetags/ralph_tags.py [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 register = Library()
 
+SENTRY_ENABLED = getattr(settings, 'SENTRY_ENABLED', False)
```
Comment (mkurek) [inline src/ralph/admin/templatetags/ralph_tags.py]: please add default values to settings for this variables and use env. variables to set value in settings

File: src/ralph/settings/base.py [hunk 2]
```diff
@@ -240,6 +244,11 @@
     'PASSWORD': os.environ.get('REDIS_PASSWORD', ''),
 }
 
+
+SENTRY_ENABLED = os_env_true('SENTRY_ENABLED'),
```
Comment (mkurek) [inline src/ralph/settings/base.py]: this will be tuple you know? :)

File: src/ralph/settings/base.py [hunk 3]
```diff
@@ -240,6 +244,11 @@
...
+SENTRY_JS_DSN = os.environ.get('SENTRY_JS_DSN', None),
+SENTRY_JS_CONFIG = os.environ.get('SENTRY_JS_CONFIG', '{}'),
```
Comment (mkurek) [inline src/ralph/settings/base.py]: shouldn't this be loaded to json?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**